databases/week1/homework/homeowrk-week1.sql

```diff
@@ -0,0 +1,51 @@
+use	 hyf_lesson1;
+
+#1 
+SELECT count(*)
+FROM task;
+
+#2
+SELECT count(*)
+FROM task
+WHERE due_date IS NOT NULL;
+
+#3
+SELECT task.*
+FROM status JOIN task ON task.status_id=status.id
+WHERE status.name = "done";
+
+#4
+SELECT task.*
+FROM status JOIN task ON task.status_id=status.id
+WHERE status.name <> "done";
+
+# QUERY 5
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+# QUERY 6
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 0, 1;
```

Comment:
This is correct, just to say that `LIMIT 1` would give you the same result. The first number (`0` in this case) is like an offset, and an offset of 0 is the same as no offset.

Reply:
Thanks, Marta, It's good to know.